hearth/spec/hearth/middleware/send_spec.rb

```diff
@@ -113,23 +125,35 @@ module Middleware
 
           context 'stub is a hash' do
             let(:stub_hash) { { param1: 'value' } }
+            let(:output_type) { StubOutput.new(**stub_hash) }
+
             before { stubs.add_stubs(operation, [stub_hash]) }
 
             it 'uses the stub class to stub the response' do
+              expect(stub_params_class).to receive(:build)
+                .with(stub_hash, context: 'stub[operation]')
+                .and_return(output_type)
```

Comment:
Why create StubOutputParams (with implementation) if you are mocking it? I'd say make a double like stub_class instead of defining StubOutputParams